package.json

```diff
@@ -9,20 +9,30 @@
   "scripts": {
     "install": "prebuild-install -r napi || node-gyp rebuild",
     "prebuild": "prebuild -t 3 -r napi --strip",
+    "precommit": "lint-staged",
```

Comment:
Not entirely sure how this works, there doesn't appear to be an install of husky on parcel core repo

Reply:
There is, but not where it should be...
https://github.com/parcel-bundler/parcel/blob/1af603e1cfab014585ded8aec4d3dcdb7e685987/packages/core/parcel-bundler/package.json#L96